src/pbt/deployment/jobs/databricks.py

```diff
@@ -937,27 +937,28 @@ def execute_job(_fabric_id, pipeline_id, config_name, config_content):
 
         return await_futures_and_update_states(futures, self._STEP_ID)
 
-    def _upload_configuration(self, fabric_id, pipeline_id, config_name, configuration_content):
-        def base_path(fab_id: Optional[str]):
-            if self.project_config.is_volume_supported(fabric_id):
-                _base_path = self.project_config.get_db_base_path(fab_id)
-            elif fab_id in self.project.fabric_volumes_detected.keys():
-                _base_path = self.project.fabric_volumes_detected[fabric_id]
-            else:
-                raise NotImplementedError("volume must either be defined in jobs or in project config (fabrics.yml)")
+    def _base_path(self, fab_id: Optional[str], pipeline_id):
+        if fab_id is None or self.project_config.is_volume_supported(fab_id):
+            _base_path = self.project_config.get_db_base_path(fab_id)
+        elif fab_id in self.project.fabric_volumes_detected.keys():
+            _base_path = self.project.fabric_volumes_detected[fab_id]
+        else:
```

Comment:
Base Path is DBFS even when volumes are supported?

Reply:
this is just the default behavior from before. IMO we should deprecate dbfs:/ as the default since DBFS has been deprecated by Databricks